pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example.local
```

Comment:
Most people will use kube-lego to automatically generate the secret I think.

https://github.com/kubernetes/charts/search?utf8=%E2%9C%93&q=secretName&type=

I do like the idea of adding a default name though, especially since kube-lego will use `service-name-tls` by default. This is how chronograf handles it:
```
secretName: {{ .Values.ingress.secretName | default (printf "%s-tls" (include "fullname" .)) }}
```